### ISSUE TITLE
Missing <h1> and alt text in main website logo

### ISSUE THREAD

**Issue by aesedepece** [witnet/website#79]
https://github.com/witnet/website/blob/4f886d63092a6c8740d0ffbc010c074ab3d4238a/components/NavBar.vue#L4-L11
